src/api/Alert/types.ts

```diff
@@ -35,7 +36,7 @@ export interface IPopulatedCaseVisualization extends ICaseVisualization {
   edges: VizEdge[];
 }
 
-export interface ICaseVisualization {
```

Comment:
Why use ICaseVisualization and not CaseVisualization?

also what's the point of extending a partial interface? everything is optional. The idea is to create a common interface both extends.